docs/criteria/connectors/rc.md

```diff
@@ -37,33 +37,37 @@ These connectors are exempt from running the TPC derived test packages, and rely
-| ------------- | - | - | - | - | - | - |
-| Clickhouse    | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
-| Databricks    | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
-| Delta Lake    | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
-| Dremio        | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
-| File          | ✅ (1) | ✅ (0.5) | ❌ | ❌ | ✅ | ❌ |
-| FTP/SFTP      | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
-| GraphQL       | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
-| GitHub        | ❌ | ❌ | ⚠️ | ❌ | ❌ | ⚠️ |
-| HTTP/HTTPS    | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
-| MS SQL        | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
-| MySQL         | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
-| ODBC          | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
-| PostgreSQL    | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
-| Sharepoint    | ❌ | ❌ | ⚠️ | ❌ | ❌ | ⚠️ |
-| Snowflake     | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
-| Spice.AI      | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
-| S3            | ✅ (1) | ✅ (0.5) | ⚠️ | ❌ | ✅ | ❌ |
-| Azure BlobFS  | ✅ (1) | ✅ (0.5) | ⚠️ | ❌ | ✅ | ❌ |
-| Spark         | ✅ (1) | ✅ (0.5) | ✅ | ✅ | ✅ | ✅ |
+| Connector | [TPC-H Derived Tests (Scale Factor)](#tpc-h) | [TPC-DS Derived Tests (Scale Factor)](#tpc-ds) | [Federation](#federation) | [Data Correctness](#data-correctness) | [Streaming](#streaming) | [Native Schema Inference](#schema-inference) |
+| - | - | - | - | - | - | - |
+| Clickhouse              | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
+| Databricks              | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
+| Delta Lake              | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
+| Dremio                  | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
+| DuckDB                  | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
+| File                    | ✅ (1) | ✅ (1) | ❌ | ❌ | ✅ | ❌ |
+| FTP/SFTP                | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
+| GraphQL                 | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
+| GitHub                  | ❌ | ❌ | ⚠️ | ❌ | ❌ | ⚠️ |
+| HTTP/HTTPS              | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
+| MS SQL                  | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
+| MySQL                   | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
+| ODBC                    | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
+| PostgreSQL              | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
+| Sharepoint              | ❌ | ❌ | ⚠️ | ❌ | ❌ | ⚠️ |
+| Snowflake               | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
+| Spice.AI Cloud Platform | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
+| S3                      | ✅ (1) | ✅ (1) | ⚠️ | ❌ | ✅ | ❌ |
+| Azure BlobFS            | ✅ (1) | ✅ (1) | ⚠️ | ❌ | ✅ | ❌ |
+| Spark                   | ✅ (1) | ✅ (1) | ✅ | ✅ | ✅ | ✅ |
 
 ### All Connectors
 
 These requirements are imposed on every connector, regardless of the selected features/tests in the above table.
 
-- [ ] [Core Arrow Data Types](../definitions.md) are supported
+The RC release criteria expand on and require that all [Beta release criteria](./beta.md) continue to pass for the connector.
+
+- [ ] All [Beta release criteria](./beta.md) pass for this connector.
```

Comment:
> All [Beta release criteria](./beta.md) pass for this connector.

I found this hard to use as diff between Beta and RC is unclear - what specific point of Beta must additionally be satisfied? It looks like RC includes everything from Beta already or I miss something?

Reply:
RC should include everything from Beta and extend on it. Beta:

* includes only TPC-H, while RC is TPC-H and TPC-DS
* does not include native schema inference, data correctness tests, and only partial coverage for federation
* excludes the requirement to print warnings instead of failing on unsuported column types

Some segments are repeated in RC. This is intended, because they should explicitly be re-tested. For example, Beta tests TPC-H at scale 1, while RC tests at 100. Documentation should continue to be updated in RC, because there may be new exceptions from RC that need to be documented. There are a few points that can be excluded from RC because they exist in Beta that I've updated to move across.